internal/film/fimlrepo_mongo.go

```diff
@@ -1,10 +1,15 @@
 package film
 
 import (
-	imdb "github.com/PhamDuyKhang/littledetective/internal/pkg/crawler"
-	"github.com/PhamDuyKhang/littledetective/internal/types"
 	"github.com/globalsign/mgo"
 	"github.com/globalsign/mgo/bson"
+
+	"github.com/PhamDuyKhang/littledetective/internal/types"
+)
+
+const (
+	DATABASE_NAME   = "imdbfilms"
+	COLLECTION_NAME = "moive"
```

Comment:
Please consider more meaningful when naming vars, consts.
E.g.: This one should should be: `ImdbDb`, `DbImdb`, `MovieCollection`. The above for Elasticsearch consts can be `ElasticImdbIndex`...

When naming vars, consts make sure it can answer the question What is this?